src/main.ts

```diff
@@ -262,6 +263,26 @@ export class Purchases {
     );
   }
 
+  /**
+   * Retrieves a specific offering by its identifier.
+   * @param offeringIdentifier - The identifier of the offering to retrieve.
+   * @param params - The parameters object to customise the offerings fetch. Check {@link GetOfferingsParams}
+   */
+  public async getOffering(
```

Comment:
I do have the concern of people using this and hardcoding the offering id instead of using the current offering... I wonder if we should also have a `getCurrentOffering` to access the current one...

Reply:
Good point! Maybe `GetOfferingParams` could contain a field that contains either `Current` or an offering identifier? Not sure what the best way to describe that is in TypeScript though, but I believe we do a similar thing for `PurchasesAreCompletedBy`. 

This would prevent yet another new offerings method.

Reply:
Yeah, we can add an optional `offeringId` field to the `GetOfferingParams` [interface](https://github.com/RevenueCat/purchases-js/blob/main/src/entities/get-offerings-params.ts#L5) and use it [here](https://github.com/RevenueCat/purchases-js/blob/main/src/main.ts#L229).

Reply:
I've given it a shot adding a new parameter to `GetOfferingsParams` named `offeringIdentifier`, which could be a string or a OfferingKeyword (an enum which only value is `Current`). Let me know what you think. I'm not a fan of the `OfferingKeyword` name, but couldn't find anything better.